app/assets/javascripts/services/event_notifications_service.js

```diff
@@ -129,6 +129,8 @@ function eventNotifications($timeout, API) {
       type: levelToType(type),
       message: message,
       data: notificationData,
+      actionTitle: __('View details'),
```

Comment:
@mzazrivec is this okay?

Reply:
Yes, I think so.

Reply:
Thanks :four_leaf_clover: :four_leaf_clover: